### DISCUSSION TITLE
Recycle Bin

### DISCUSSION THREAD

**Discussion by NaveedAhmadHematmal** [VDTS/CodeX.VaxineSolution#87] [Ideas]
What if we implement a `Recycle Bin` that holds deleted data for a while, and we can restore the data back within the time.

**Comment by YasinZahin** [comment]
Good idea.

**Comment by SaeedaRasuly** [comment]
Agree, That would be plus point for the app

**Reply by AbdulBasirZafar** [reply]
I have seen it in some apps and it is useful so it is good if we implement it

**Comment by NaveedAhmadHematmal** [comment]
Recycle bin will keep data for 15 days. In these 15 days, data should be deleted or restored otherwise Recycle Bin will auto-delete it. There will be a button to `Empty the recycle bin` that will delete everything from recycle bin.

**Reply by SaeedaRasuly** [reply]
15 days till a month is ok nor more nor less

**Comment by YasinZahin** [comment]
Why 15 days? Two or three days will be better.

**Reply by NaveedAhmadHematmal** [reply]
@SaeedaRasuly! @YasinZahin is right, user should decide whether to delete or restore an item within a vaccine period. So 3-5 days are good.

**Reply by SaeedaRasuly** [reply]
@NaveedAhmadHematmal okk so for this reason 3-5 days are ok
